File: C++/1015/1015.cpp
#include <cmath>
#include <cstdio>
#include <iostream>
using namespace std;

int main() {
	double x1, y1, x2, y2, distance;

	cin >> x1 >> y1;
	cin >> x2 >> y2;

	distance = sqrt(((x2-x1)*(x2-x1)) + ((y2-y1)*(y2-y1)));

	printf("%.4f\n", distance);
}

File: Pascal/1015/1015.pas
var
	x1, y1 : double;
	x2, y2 : double;
	distance : double;
begin
	readln(x1, y1);
	readln(x2, y2);

	distance := Sqrt(((x2-x1)*(x2-x1)) + ((y2-y1)*(y2-y1)));

	writeln(distance:4:4);
end.
